### PR TITLE
refactor: post tag inspector

### PR DESCRIPTION
**Pull request by guanbinrui**
## Description

Disable post replacement and inject a tag inspector instead.

### Usage

```ts
const sns: Plugin.SNSAdaptor.Definition {
    PostTagInspector: function TagInspector(props) {
        return <Typography>HELLO WORLD</Typography>
    },
}
```

## Type of change

- [ ] Documentation
- [x] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

### CODE REVIEW

File: packages/plugin-infra/src/types.ts [hunk 1]
```diff
@@ -996,6 +983,11 @@ export enum CurrentSNSNetwork {
     Minds = 4,
 }
 
+export enum TagType {
+    CASH = 1,
+    HASH = 2,
```
Comment (Jack-Works) [inline packages/plugin-infra/src/types.ts:988]: Add mention and topic. `@SomeOne`, `#topic`

File: packages/mask/src/social-network-adaptor/twitter.com/injection/PostTagInspector.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        .startWatch(
+            {
+                childList: true,
+                subtree: true,
+            },
+            signal,
+        )
+    startWatch(watcher, signal)
+    createReactRootShadowed(watcher.firstDOMProxy.afterShadow, { signal }).render(
+        <PostTagInspector type={TagType.CASH} name="USDT" />,
```
Comment (Jack-Works) [inline packages/mask/src/social-network-adaptor/twitter.com/injection/PostTagInspector.tsx:21]: why hard-coded USDT?

File: packages/plugin-infra/src/types.ts [hunk 3]
```diff
@@ -410,6 +410,11 @@ export namespace Plugin.SNSAdaptor {
         PostInspector?: InjectUI<{}>
         /** This UI will be rendered for action of each post found. */
         PostActions?: InjectUI<{}>
+        /** This UI will be rendered as tabs on the profile card */
+        PostTagInspector?: InjectUI<{
+            type: TagType
+            name: string
+        }>
```
Comment (Jack-Works) [inline packages/plugin-infra/src/types.ts:417]: Extract a PostTagInspectorProps.

**Review by Jack-Works — CHANGES_REQUESTED** [review]

File: packages/mask/src/social-network-adaptor/twitter.com/injection/PostTagInspector.tsx (hunk 2 above)
Comment (guanbinrui) [reply packages/mask/src/social-network-adaptor/twitter.com/injection/PostTagInspector.tsx:21]: Testing only.

File: packages/plugin-infra/src/types.ts (hunk 3 above)
Comment (guanbinrui) [reply packages/plugin-infra/src/types.ts:417]: Will do.

**Pull request closed without merging**